test/hotspot/jtreg/compiler/loopopts/Test8335708.java

```diff
@@ -0,0 +1,69 @@
+/*
```

Comment:
Can you rename the test file? I think the new common practice is to give it a descriptive name, rather than just the bug number which is already tracked under `@bug 8335708` anyway ;)

Reply:
Done. I think it's more explicit.